text_generation/causal_lm/cpp/continuous_batching/library/src/sampler.hpp

```diff
@@ -203,122 +205,6 @@ class GroupBeamSearcher {
 };
 
 using LogitWithIdx = std::pair<float, size_t>;
```

Comment:
BTW, it's the same as `Token`
See https://github.com/ilya-lavrenov/openvino.genai/blob/bfaa55a1ecf28ae7d4970a480448ef427730edc6/text_generation/causal_lm/cpp/continuous_batching/library/src/sampler.hpp#L64-L67